src/check_release.rs

```diff
@@ -219,109 +310,17 @@ pub(super) fn run_check_release(
-            }
-
-            config
-                .log_info(|config| {
-                    writeln!(
-                        config.stdout(),
-                        "{}Failed in:{}",
-                        Style::new().bold(),
-                        Reset
-                    )?;
-                    Ok(())
-                })
-                .expect("print failed");
+            config.log_info(|config| {
+                writeln!(
+                    config.stdout(),
+                    "\n--- failure {}: {} ---\n",
+                    &semver_query.id,
+                    &semver_query.human_readable_name
+                )?;
+                Ok(())
+            })?;
 
-            for semver_violation_result in results {
-                let pretty_result: BTreeMap<Arc<str>, TransparentValue> = semver_violation_result
-                    .into_iter()
-                    .map(|(k, v)| (k, v.into()))
-                    .collect();
-
-                if let Some(template) = semver_query.per_result_error_template.as_deref() {
-                    let message = config
-                        .handlebars()
-                        .render_template(template, &pretty_result)
-                        .context("Error instantiating semver query template.")
-                        .expect("could not materialize template");
-                    config
-                        .log_info(|config| {
-                            writeln!(config.stdout(), "  {}", message)?;
-                            Ok(())
-                        })
-                        .expect("print failed");
-
-                    config
-                        .log_extra_verbose(|config| {
-                            let serde_pretty =
-                                serde_json::to_string_pretty(&pretty_result).expect("serde failed");
-                            let indented_serde = serde_pretty
-                                .split('\n')
-                                .map(|line| format!("    {line}"))
-                                .join("\n");
-                            writeln!(
-                                config.stdout(),
-                                "\tlint rule output values:\n{}",
-                                indented_serde
-                            )?;
-                            Ok(())
-                        })
-                        .expect("print failed");
-                } else {
-                    config
-                        .log_info(|config| {
-                            writeln!(
-                                config.stdout(),
-                                "{}\n",
-                                serde_json::to_string_pretty(&pretty_result)?
-                            )?;
-                            Ok(())
-                        })
-                        .expect("print failed");
-                }
-            }
+            print_failed_lint(config, semver_query, results)?;
```

Comment:
Corresponding change to the function signature change above.
```suggestion
            print_failed_lint(config, semver_query, &results)?;
```